core-services/prow/02_config/stolostron/governance-policy-propagator/_prowconfig.yaml

```diff
@@ -101,6 +101,21 @@ branch-protection:
                 - SonarCloud Code Analysis
                 - Upstream reference checks
                 - Red Hat Konflux / governance-policy-propagator-acm-212-on-pull-request
+            release-2.13:
+              protect: true
+              required_pull_request_reviews:
+                dismiss_stale_reviews: true
+                required_approving_review_count: 1
+              required_status_checks:
+                contexts:
+                - KinD tests (minimum)
+                - KinD tests (latest)
+                - KinD / Tests (latest, true)
+                - KinD / Tests (latest, false)
+                - KinD / Tests (latest, false, hosted)
+                - SonarCloud Code Analysis
+                - Upstream reference checks
+                - Red Hat Konflux / governance-policy-propagator-acm-212-on-pull-request
```

Comment:
This (and others) needs to have `213` rather than `212`:
```suggestion
                - Red Hat Konflux / governance-policy-propagator-acm-213-on-pull-request
```